WooCommerce/WooCommerceTests/Yosemite/StoresManagerTests.swift

```diff
@@ -173,7 +174,13 @@ extension DefaultStoresManager {
     /// Returns a StoresManager instance with testing Keychain/UserDefaults
     ///
     static var testingInstance: DefaultStoresManager {
-        return DefaultStoresManager(sessionManager: .testingInstance)
+        return DefaultStoresManager(concreteSessionManager: .testingInstance)
+    }
+
+    /// Allow initializing a test `DefaultStoresManager` using a concrete implementation of `SessionManager`
+    ///
+    convenience init(concreteSessionManager: SessionManager) {
+        self.init(sessionManager: concreteSessionManager)
     }
```

Comment:
It doesn't look like we need the `init(concreteSessionManager:)`. Looks like this works just fine:

```suggestion
    static var testingInstance: DefaultStoresManager {
        DefaultStoresManager(sessionManager: SessionManager.testingInstance)
    }
```

Out of scope: I'm worried about this `static` instance. This means that we're keeping this object until all the tests have finished running. 😬

Reply:
>It doesn't look like we need the init(concreteSessionManager:). Looks like this works just fine:

Yeah, I just got too excited trying to preserve the `.testingInstance` syntactic sugar 😜 

Fixed in https://github.com/woocommerce/woocommerce-ios/pull/3305/commits/5d4e4e962ce69fdb4fa1133d9b72e90bc5e57374

> I'm worried about this static instance

Is this a static instance? I thought it would be functionally equivalent to `static func getTestingInstance() -> DefaultStoresManager` – the value only lives the lifetime of the test, then each test gets a new version because it's a `getter` rather than an actual variable?

Reply:
You're right. Yeah, I can't read Swift. 😅